src/ai/backend/web/proxy.py

```diff
@@ -141,13 +141,13 @@ async def decrypt_payload(request):
 
 async def web_handler(request, *, is_anonymous=False) -> web.StreamResponse:
     path = request.match_info.get('path', '')
-    name = request.match_info.get('name', '')
+    first_path = request.path.strip(r'\/').split('/')[0]  # extract the first path
```

Comment:
```suggestion
    first_path = request.path.lstrip('/').partition('/')[0]  # extract the first path
```
- Simplify the string pattern and use `lstrip()` only
- `str.partition()` works even when the trailing part does not contain any slash.

Reply:
it's more safer than the previous change. I'll apply it right away!